types/query-helpers.d.ts

```diff
@@ -3,6 +3,7 @@ import {waitForOptions} from './wait-for'
 
 export interface SelectorMatcherOptions extends MatcherOptions {
   selector?: string
+  ignore?: string | boolean
```

Comment:
Perhaps this would help with code completion? I'm not sure if it's `'script'` or `'script, style'` though, the docs are inconsistent.
```suggestion
  ignore?: 'script' | string | boolean
```

Reply:
The default value is 'script, style' if that's what you're looking for:
https://github.com/testing-library/dom-testing-library/blob/master/src/config.ts#L44

Reply:
Ah, I thought TypeScript already supported that, as I saw it in use elsewhere